doc/classes/Color.xml

```diff
@@ -4,8 +4,8 @@
 		Color in RGBA format with some support for ARGB format.
 	</brief_description>
 	<description>
-		A color is represented as red, green and blue (r,g,b) components. Additionally, "a" represents the alpha component, often used for transparency. Values are in floating point and usually range from 0 to 1. Some methods (such as set_modulate(color)) may accept values &gt; 1.
-		You can also create a color from standardised color names with Color.ColorN (e.g. Color.green) or [method @GDScript.ColorN].
+		A color is represented by red, green, and blue [code](r, g, b)[/code] components. Additionally, [code]a[/code] represents the alpha component, often used for transparency. Values are in floating point and usually range from 0 to 1.  Some properties (such as [member CanvasItem.modulate] ) may accept values &gt; 1.
```

Comment:
You have a double space between the third and fourth sentences.

You also have a unneeded space just after the link to `CanvasItem.modulate`.

Reply:
Good catch, thanks.